getting_started.md

```diff
@@ -104,17 +142,19 @@ plt.show()
 ```
 
 It's obvious that there's something unusual about the trees in the middle of this
-chromosome, where the selective sweep occurred. It's not particularly efficient to pull
-out a tree from the middle of a tree sequence, but it *can* be done, via the
-{meth}`TreeSequence.at()<tskit.TreeSequence.at>` method. Here's the tree at the location
-of the sweep, drawn using the  {meth}`Tree.draw_svg()<tskit.Tree.draw_svg>` method
-(see the {ref}`visualization tutorial <sec_tskit_viz>` for more visualization
-possibilities):
+chromosome, where the selective sweep occurred. 
+
+Currently, it's not particularly efficient to pull out individual trees from the middle
```

Comment:
I'm wondering here if we should suggest people who have use cases for random access to comment in say https://github.com/tskit-dev/tskit/issues/684 so that we can know. Generally mentioning occasionally in the docs that we welcome issues/discussions seems a good thing to reduce the barriers.

Reply:
Yes, I would agree with that, as long as we note in that issue to take the comment out of the tut when the issue is closed.

Reply:
Should I do this now, before you merge it @benjeffery ?